src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -608,6 +608,12 @@ class TypeSpec private constructor(builder: TypeSpec.Builder) {
 
     @JvmStatic fun expectClassBuilder(className: ClassName) = expectClassBuilder(className.simpleName())
 
+    @JvmStatic fun externalClassBuilder(name: String) = Builder(Kind.CLASS, name, null).apply {
```

Comment:
I wonder if factory methods like these are really needed...  We allow `external` on interfaces and objects, but we don't have specific factory methods for those, and it would've been too much anyway. `external class` is undoubtedly a more popular use-case than `external interface`, but does it warrant having such a specific factory method? We might think about removing `expectClassBuilder()` as well...

Reply:
I would very much prefer not having specific factory methods!

Reply:
Should I remove all of them or just don't add the `external` ones?

Reply:
In scope of this PR, just don't add new ones